src/foam/comics/DAOControllerView.js

```diff
@@ -87,48 +115,83 @@ foam.CLASS({
             this.importedUpdateView :
             { class: 'foam.comics.DAOUpdateControllerView' };
       }
-    },
-    {
-      class: 'String',
-      name: 'title',
-      expression: function(data$data$of) {
-        return 'Browse ' + data$data$of.name;
-      }
     }
   ],
 
   reactions: [
-    [ 'data', 'action.create', 'onCreate' ],
-    [ 'data', 'edit', 'onEdit' ],
-    [ 'data', 'action.findRelatedObject', 'onFindRelated' ],
-    [ 'data', 'finished', 'onFinished' ],
-    [ 'data', 'export', 'onExport' ]
+    ['data', 'action.create', 'onCreate'],
+    ['data', 'edit', 'onEdit'],
+    ['data', 'action.findRelatedObject', 'onFindRelated'],
+    ['data', 'finished', 'onFinished'],
+    ['data', 'export', 'onExport']
   ],
 
   methods: [
     function initE() {
       var self = this;
 
       this.data.border.add(
-        this.E().addClass(this.myClass()).
-        start().
-          hide(self.data.searchHidden$).
-          show(self.data.filtersEnabled$).
-          add(self.cls.PREDICATE).
-        end().
-        start().
-          style({ 'overflow-x': 'auto' }).
-          start().
-            addClass('actions').
-            show(self.mode$.map((m) => m === foam.u2.DisplayMode.RW)).
-              start().add(self.cls.getAxiomsByClass(foam.core.Action)).end().
-          end().
-          start().
-            style({ 'overflow-x': 'auto' }).
-            tag(this.summaryView, { data$: this.data.filteredDAO$ }).
-          end().
-        end()
-      );
+        this.E()
+          .addClass(this.myClass())
+          .start()
+            .addClass(this.myClass('top-row'))
+            .start()
+              .addClass(this.myClass('title-container'))
+              .start('h1')
+                .add(this.data.title$)
+              .end()
+              .add(this.data.subtitle$)
+            .end()
+            .callIf(this.data.primaryAction, function() {
```

Comment:
Cleaner to call .callIfElse instead.

commit 6a1883c34a4e262a74e2b276cc1a4f374378e23b (HEAD -> master, origin/master, origin/HEAD)
Author: Kevin Greer <kgrgreer@gmail.com>
Date:   Thu Nov 1 09:41:05 2018 -0400

    Add callIfElse to Element.

Reply:
Resolved by 5a50e09.